sparkle/types/features.py

```diff
@@ -0,0 +1,82 @@
+"""Types definition for features to unify extractors across the Sparkle platform."""
+from __future__ import annotations
+from enum import Enum
+
+
+class FeatureGroup(str, Enum):
```

Comment:
These are SATZilla specific types, I think it's better if these are abstract classes that are filled separately by each extractor (like the extractor wrapper)

Reply:
We can write a ticket about that